From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Mon, 16 Mar 2026 10:31:00 +0800
Subject: man-nocaseglob

# DP: Clarify documentation about case-insensitive pathname expansion

# DP: Clarify documentation about case-insensitive pathname expansion
---
 doc/bash.1 | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/doc/bash.1 b/doc/bash.1
index ef60897..db1a957 100644
--- a/doc/bash.1
+++ b/doc/bash.1
@@ -4376,6 +4376,10 @@ If the shell option
 .B nocaseglob
 is enabled, the match is performed without regard to the case
 of alphabetic characters.
+Note that when using range expressions like
+[a-z] (see below), letters of the other case may be included,
+depending on the setting of
+.B LC_COLLATE.
 .PP
 When a pattern is used for pathname expansion,
 the character
